code/modules/modular_computers/file_system/programs/command/comm.dm

```diff
@@ -312,3 +292,131 @@ var/last_message_id = 0
 
 /datum/comm_message_listener/proc/Remove(var/list/message)
 	messages -= list(message)
+/*
+Command action procs
+*/
+/proc/post_status(var/command, var/data1, var/data2)
```

Comment:
A small nitpick since we're here already. This could use a clearer name perhaps?